components/costs/README.md

```diff
@@ -0,0 +1,7 @@
+# Cost and usage reports
```

Comment:
Please use the existing convention of `superwerker.yaml` and include this component using feature flippers.